.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "devcontainer-fluent-bit",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
```

Comment:
What about Windows dev containers? That would be useful too I think

Reply:
I changed it with fluent-bit debug image.